tests/kernel/timer/timer_behavior/pytest/test_timer.py

```diff
@@ -62,13 +65,13 @@ def do_analysis(test, stats, stats_count, config, sys_clock_hw_cycles_per_sec):
                 f', "min_bound_us":{min_bound * 1_000_000:.6f}'
                 f', "max_bound_us":{max_bound * 1_000_000:.6f}'
                 f', "expected_period_cycles":{expected_period:.0f}'
+                f', "max_stddev":{max_stddev:.6f}'
```

Comment:
```suggestion
                f', "MAX_STD_DEV":{max_stddev:.6f}'
```
record field name should be the same if it is the same value with jitter_drift.c

Reply:
done